### DISCUSSION TITLE
What does the "reverse" option actually do?

### DISCUSSION THREAD

**Discussion by l2aelba** [inokawa/virtua#457] [Q&A] [answered]
I just tested the basic chat app layout from ([example](https://inokawa.github.io/virtua/?path=/story/advanced-chat--default)).

From the doc:

> If true, items are aligned to the end of the list when total size of items are smaller than viewport size. It's useful for chat like app.

This is just pushed the items to the bottom right?

Could you explain this further? Am I missing something?

**Comment by inokawa** [comment, marked as answer]
> This is just pushed the items to the bottom right?

Yes. There are no effect not mentioned and it's for usecases like #283 .
